networks/local/blockchain-local/Dockerfile

```diff
@@ -73,9 +74,9 @@ RUN go build -mod vendor -tags=cleveldb,dynamic -ldflags '-w -s -X github.com/co
 ## Run provenance for ARM
 FROM debian:buster-slim as provenance-arm64
 ENV LOCALNET=1
-# This is for M1 to find librdkafka
-ENV PKG_CONFIG_PATH="/usr/local/lib"
 ENV LD_LIBRARY_PATH="/usr/local/lib"
```

Comment:
can we remove LD_LIBRARY_PATH on line 78 since it's already defined on line 25?

Reply:
I'm not fully versed in `Dockerfile`, but since it's after a new `FROM` line, line 25 might not apply here?

Reply:
Yeah, they are separate build stages so both are needed. One stage is used for building x86 and the other is for arch64.

Reply:
> Yeah, they are separate build stages so both are needed. One stage is used for building x86 and the other is for arch64.

👍 unfortunately i opened this PR, so can't really approve it :(